src/test/java/tests/ActionsWithProductsTest.java

```diff
@@ -0,0 +1,139 @@
+package tests;
+
+import org.junit.Test;
+import org.testng.Assert;
+import pages.HomePage;
+import pages.LoginPage;
+import pages.ProductsPage;
+
+public class ActionsWithProductsTest extends BaseTest {
+
+    @Test
+    public void productActionCartTest() {
+        HomePage homePage = new HomePage(getDriver());
+        ProductsPage productsPage = new ProductsPage(getDriver());
+
+        homePage.openHomePage();
+        homePage.navigateToCategoriesFromMainPage();
+        homePage.navigateToLaptopCategory();
+        productsPage.putFirstProductInCart();
+        productsPage.clickToHeaderCartButton();
+        Assert.assertEquals(productsPage.getFirstProductName(), productsPage.getProductNameInCart());
+    }
+
+    @Test
+    public void productKitActionCartTest() {
+        HomePage homePage = new HomePage(getDriver());
+        ProductsPage productsPage = new ProductsPage(getDriver());
+
+        homePage.openHomePage();
+        homePage.navigateToCategoriesFromMainPage();
+        homePage.navigateToLaptopCategory();
+        productsPage.clickToFirstProduct();
```

Comment:
all necessary pages can be created once at the top of class and used in all tests